### ISSUE TITLE
Update to go modules

### ISSUE THREAD

**Issue by LucasRoesler** [openfaas/faas-swarm#72]
## Context

`dep` is officially being deprecated , the repo is already archived https://github.com/golang/dep 

I have already tried the initial update but ran into several issues.  It seems like we need to wait for this https://github.com/containerd/containerd/issues/3031 to be resolved, which is currently blocked on this PR https://github.com/containerd/cri/pull/1377#issuecomment-664136849 So progress is being made upstream.

**Comment by mikebrow** [comment]
containerd/cri has been merged into containerd/containerd on master... FYI

**Comment by alexellis** [comment]
Thank you for the update. Is there a stable release available as of yet?
